api/lib/source/cosmosV0-source.js

```diff
@@ -310,25 +310,32 @@ class CosmosV0API extends RESTDataSource {
               title: `Deposit Period Ended`,
               // the deposit period can end before the time as the limit is reached already
               time:
+                proposal.voting_start_time !== `0001-01-01T00:00:00Z` &&
```

Comment:
what is that weird date?

Reply:
so time = undefind

Reply:
aah. Wow :eyes: Javascript can be very crazy